lib/matplotlib/animation.py

```diff
@@ -1313,15 +1311,15 @@ class TimedAnimation(Animation):
 
     repeat_delay : number, optional
         If the animation in repeated, adds a delay in milliseconds
-        before repeating the animation.  Defaults to `None`.
+        before repeating the animation.  Defaults to ``None``.
```

Comment:
I tend to prefer two spaces after a period (https://en.wikipedia.org/wiki/Sentence_spacing -- I know there are arguments both ways).  I don't think it's worth changing the style throughout the docs, but at least locally it should be better to uniformize it (so here I settled towards two-spacing).  Same reply applies to comments below.